src/typeahead/typeahead.ts

```diff
@@ -57,7 +57,8 @@ export class NgbTypeahead implements OnInit,
   private _popupService: PopupService<NgbTypeaheadWindow>;
   private _positioning = new Positioning();
   private _valueChanges = new Subject<string>();
-  private _windowRef: ComponentRef<NgbTypeaheadWindow>;
+
+  windowRef: ComponentRef<NgbTypeaheadWindow>;
```

Comment:
Ah, it is used in host... Actually I'm not sure what the rule for the `host` should be :-)